ArrtSource/Rendering/UI/StartSessionDlg.ui

```diff
@@ -210,7 +213,7 @@
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
-        <string>Max Time (H:MM):</string>
+        <string>Max Session Duration (H:MM):</string>
```

Comment:
Will this be narrated correctly? You wrote 'H colon MM' above, but now it is 'H:MM'. Both look fine to me, but I would like it to be consistent.

Reply:
This is the label, it won't be narrated at all. The edit box right next to it is the one that gets focus and is narrated, that's why I used the 'colon' there.